ui/appui-layout-react/src/appui-layout-react/state/NineZoneStateReducer.ts

```diff
@@ -530,9 +530,31 @@ export function NineZoneStateReducer(
       if (location && isPopoutTabLocation(location)) return state;
 
       const savedTab = state.savedTabs.byId[id];
+
+      const tab = state.tabs[id];
+
+      let contentHeight = 800;
+      let contentWidth = 600;
+
+      if (tab.preferredFloatingWidgetSize) {
+        contentWidth = tab.preferredFloatingWidgetSize.width;
+        contentHeight = tab.preferredFloatingWidgetSize.height;
+      } else {
+        const popoutContentContainer = document.getElementById(
+          `content-container:${savedTab?.id}`
```

Comment:
If the `savedTab` wasn't found, this is when we'll actually want to get the container size, and if it's not found, then the id will be `undefined`, as this is the `id` anyway and we are doing a `getElementById`, might as well simply use the provided `id`.
```suggestion
          `content-container:${id}`
```